src/ReactDOM.purs

```diff
@@ -43,35 +38,23 @@ renderToStaticMarkup :: ReactElement -> String
 renderToStaticMarkup = runFn1 renderToStaticMarkupImpl
 
 foreign import renderImpl
-  :: forall eff
-   . EffFn4
-       (dom :: DOM | eff)
+  :: EffectFn4
        (Maybe ReactComponent)
        (ReactComponent -> Maybe ReactComponent)
        ReactElement
        Element
        (Maybe ReactComponent)
 
 foreign import unmountComponentAtNodeImpl
-  :: forall eff
-   . EffFn1
-       (dom :: DOM | eff)
+  :: EffectFn1
        Element
        Boolean
 
 foreign import findDOMNodeImpl
-  :: forall eff
-   . EffFn1
-       (dom :: DOM | eff)
+  :: EffectFn1
        ReactComponent
        Element
 
 foreign import renderToStringImpl :: Fn1 ReactElement String
 
 foreign import renderToStaticMarkupImpl :: Fn1 ReactElement String
-
-refToNode :: Nullable Ref -> Maybe Node
-refToNode ref = toMaybe (coerce ref)
-  where
-  coerce :: Nullable Ref -> Nullable Node
-  coerce = unsafeCoerce
```

Comment:
I'm not sure why this function was here and exported since it is unsafe and unnecessary. It's much better to just go through Foreign and use Web.DOM's readers.